kettle/update.py

```diff
@@ -38,31 +41,35 @@ def call(cmd):
 
 
 def main():
-    call('time python3 make_db.py --buckets buckets.yaml --junit --threads 32')
+    call('time python3 make_db.py --buckets buckets.yaml --junit --threads {THREADS}')
 
-    bq_cmd = 'bq load --source_format=NEWLINE_DELIMITED_JSON --max_bad_records=1000'
+    bq_cmd = 'bq load --source_format=NEWLINE_DELIMITED_JSON --max_bad_records={MAX_BAD_RECORDS}'
     mj_cmd = 'pypy3 make_json.py'
 
     mj_ext = ''
     bq_ext = ''
     try:
-        call(mj_cmd + ' --days 1 --assert-oldest 1.9')
+        call(mj_cmd + ' --days 1 --assert-oldest {DAYS_OLD}')
```

Comment:
Missing an f on the second string, or could also be
```suggestion
        call(f'{mj_cmd} --days 1 --assert-oldest {DAYS_OLD}')
```